### ISSUE TITLE
Extremly slow

### ISSUE THREAD

**Issue by MTRNord** [LibreTime/libretime#503]
Hi I use this with the default install except for Apache2. I use nginx instead. I do have the problem that loading times do take 1-5minutes each action. This feels really slow. Also the action needs pretty long to get down to the actual running stream. 

I dont know what Information you need so feel free to ask. Or give any suggestions

**Comment by MTRNord** [comment]
To note: http://[URL]/?config is lightning fast

**Comment by MTRNord** [comment]
But as soon as I try to load the main site it does a time out error

**Comment by Robbt** [comment]
Yeah, ?config doesn't use Zend at all. So I suspect it is an issue with Zend1 autoloader.

**Comment by MTRNord** [comment]
So from digging into the deep hole of stackoverflow awnsers it seems like that it might be because Zend tries to open a lot of files at once. This is obviusly slow. And me running on a cloud server which restricts many file openings at once this might be the reason. I guess I have to lift the limit of that on my system and might be a bit faster

**Comment by MTRNord** [comment]
One stackoverflow suggested to run `composer dump-autoload -o --no-dev` But honestly I do not expect it to change anything

**Comment by MTRNord** [comment]
Also it seems like I am missing php caching. So lets see how much caching helps

**Comment by MTRNord** [comment]
@Robbt not sure if that is a bug or wanted but it seems that libretime never gets cached. Is that wanted or a bug? feels a bit wrong to not cache

**Comment by MTRNord** [comment]
My VPS is pretty small (1CPU and 1,5GB RAM and a 50GB SSD). Nginx is simply used because everything else uses nginx on the server. Didnt make sense to have 2 webservers for serving some php. But no I did sadly not find a solution yet :/ tweaking some timeouts just got it a few mili seconds faster and the stackoverflow suggestion had 0 results.

**Comment by Robbt** [comment]
Ok, so you are running this on a webserver that is also hosting other websites ? In general this isn't recommended because LibreTime runs a lot of services and isn't configured to work with other sites etc. Those specs would probably be sufficient but it's probably just a configuration thing with nginx.
